### DISCUSSION TITLE
Install?

### DISCUSSION THREAD

**Discussion by Jimmytheprofessionaldumbass** [monkeytypegame/monkeytype#3831] [Ideas] [answered]
I really like monkeytype, I thought of installing it but when I attempted to run the INSTALL.cmd file, if failed. I assume that file would probably be for a server or something? So is there a way to install and use monkeytype offline? If i was wrong, and that was actually the file to install monkeytype on desktop, I still need help.

**Comment by Miodec** [comment, marked as answer]
If you are using Chrome you can install the website as a PWA - go to Monkeytype and click this icon in the address bar.
<img width="49" alt="image" src="https://user-images.githubusercontent.com/13181393/207581363-d47e4faa-a63f-422b-bc1e-525c9cc67c6a.png">
